client/codegen_pinvoke/Main.cs

```diff
@@ -91,16 +92,20 @@ static int Main(string[] args)
                     lib.output_dir.Delete(true);
                 }
 
+                // TODO: move this string somewhere more sensible
+                const string single_lib_name = "constellation";

```

Comment:
keep it as `unity_states` for now, lets make renaming a separate PR (I'm working on this as part of breaking up the single dylib). Otherwise things are gonna be quite messy